src/components/Button.js

```diff
@@ -0,0 +1,13 @@
+const Button = ({ text, disabled, createToken, width }) => {
+  return (
+    <button
+      className={`${width} bg-yellow-400 text-black text-sm font-semibold uppercase px-4 py-4 rounded-full font-button hover:bg-transparent outline outline-yellow-400 hover:outline-yellow-400 focus:outline-4`}
```

Comment:
I like that you've created a button component to reuse. If you have `px-4` and `py-4`, you can just use `p-4`. But I wonder if it looks a little tall, maybe `py-3` or 2.5 would look better? And `text-md`?

Reply:
Thanks for your suggestion! I´ll try that out 😃